src/main/scala/com/linkedin/feathr/offline/job/FeatureTransformation.scala

```diff
@@ -818,6 +827,17 @@ private[offline] object FeatureTransformation {
          *
          * Note: within each FeatureGroupingCriteria, we should group by and merge extractors of the same class.
          */
+        // preprocessedDfMap is in the form of feature list of an anchor separated by comma to their preprocessed DataFrame.
```

Comment:
Do we have a test case for the change here?

Reply:
we have an integration test and a scala unit test.